whelktool/scripts/cleanups/2021/04/lxl-3547-link-expressionOf.groovy

```diff
@@ -288,11 +376,15 @@ class Norm {
         def t = normalize(title.findAll {it.key in TITLE_KEYS})
         
         // Some records have the language in the title e.g. "Tusen och en natt. Svenska" drop the language part
+        // These should normally already have been moved by moveLanguagesFromTitle()
         t.mainTitle?.with { String m ->
             def s = m.split(' ')
             if (s.last() in languageNames) {
                 t.mainTitle = normalize(s.dropRight(1).join(' '))
             }
+            if (s.size() > 3 && s[-1] in languageNames && s[-2] == "&" && s[-3] in languageNames) {
```

Comment:
What about languages conjoined with "och", "and" etc.? Is it that moveLanguagesFromTitle() never fails to handle those?

Reply:
I think this whole section should be deleted. It's a thing from before languages were moved from titles.
Will just log instad